### ISSUE TITLE
How to disable `pa_encoding_from_string` feature?

### ISSUE THREAD

**Issue by thejpster** [jnqnfe/pulse-binding-rust#7]
I'm on Ubuntu 18.04 which comes with PulseAudio 11 as best as I can tell. I'm trying to write something that uses `pulse-binding-simple`, but I can't work out how to disable the `pa_encoding_from_string` feature which is on by default. I can disable it for `pulse-binding` but it seems to be enabled again as soon as I try to use `pulse-simple`.

**Comment by jnqnfe** [comment]
Sorry for the oversight, I have added the feature flag to the `simple` and `mainloop-glib` crates to extend control of it and published the change in v2.2.2, which hopefully will fix that for you :)

**Comment by thejpster** [comment]
Do you have an example of how this would work? I tried `default-features = false` on the simple crate, but the feature was still enabled.

See https://gist.github.com/thejpster/92d791374f8f20c2ae42a4cb10e4dfe2, using

```
[dev-dependencies.libpulse-binding]
version = "2.2"
default-features = false

[dev-dependencies.libpulse-simple-binding]
version = "2.2"
default-features = false
```

**Comment by jnqnfe** [comment]
no I don't sorry, and I'm a little too busy just at the moment to investigate, so appologies in any delay if a further fix is needed

oh, you could try setting `features` to an empty set instead of the `default-features` mech

```
features = []
```

**Comment by jnqnfe** [comment]
Of course there is also always the option of filing a bug report against the package to get the maintainer to backport the symbol file fix :)

**Comment by thejpster** [comment]
Ok, I've spent quite a bit of time on this and I can't get these bindings to build. If I try and disable the default flags in the libraries I'm using, using my App's `Cargo.toml`, the flag remains enabled - seemingly via some dependency chain I can't control.

I've proposed #8, which disables the flag by default, allowing people to opt-in if they need it and they've got an OS that supports it.

**Comment by jnqnfe** [comment]
@thejpster Apologies for the delay. I managed to find a few minutes yesterday to look at this more, and I believe I have fixed it now in v2.2.3.

**Comment by thejpster** [comment]
Thanks, v2.2.3 works for me.
